packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -230,5 +230,12 @@ export const messages = {
   table_header_project_path: 'PROJECT PATH',
   table_header_type: 'TYPE',
   table_header_full_name: 'FULL NAME',
-  table_header_state: 'STATE'
+  table_header_state: 'STATE',
+  table_no_results_found: 'No results found',
+  table_title_deployed_source: 'Deployed Source',
+  table_title_deploy_errors: 'Deploy Errors',
+  table_title_pushed_source: 'Pushed Source',
+  table_title_push_errors: 'Push Errors',
+  push_conflicts_error:
+    "We couldn't complete the push operation due to conflicts. Verify that you want to keep the local versions, then run SFDX: Push Source to Default Scratch Org and Override Conflicts."
```

Comment:
@ruthemmanuelle Added an error message for source:push conflicts that closely resembles the one in the CLI. Main different is instead of telling the user to run `sfdx force:source:push -f` it specifies the corresponding VSCode command

Reply:
How about this, @brpowell ? 

We couldn’t push your source due to conflicts. Make sure that you want to overwrite the metadata in your org with your local files, then run "SFDX: Push Source to Default Scratch Org and Override Conflicts".